tests/cmds/test_users.py

```diff
@@ -153,3 +161,44 @@ def test_list_users_when_given_excluding_active_and_inactive_uses_active_equals_
     cli_state.sdk.users.get_all.assert_called_once_with(
         active=None, org_uid=None, role_id=None
     )
+
+
+def test_add_user_role_adds(
+    runner, cli_state, get_user_id_success, get_available_roles_success
+):
+    command = ["users", "add-role", "test.username@example.com", "Customer Cloud Admin"]
+    runner.invoke(cli, command, obj=cli_state)
+    cli_state.sdk.users.add_role.assert_called_once_with(TEST_USER_ID, TEST_ROLE_NAME)
+
+
+def test_add_user_role_raises_error_when_role_does_not_exist(
+    runner, cli_state, get_user_id_success, get_available_roles_success
+):
+    command = ["users", "add-role", "test.username@example.com", "test"]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 1
+    assert "Role with name test not found." in result.output
+
+
+def test_remove_user_role_removes(
+    runner, cli_state, get_user_id_success, get_available_roles_success
+):
+    command = [
+        "users",
+        "remove-role",
+        "test.username@example.com",
+        "Customer Cloud Admin",
+    ]
+    runner.invoke(cli, command, obj=cli_state)
+    cli_state.sdk.users.remove_role.assert_called_once_with(
+        TEST_USER_ID, TEST_ROLE_NAME
+    )
+
+
+def test_remove_user_role_raises_error_when_role_does_not_exist(
+    runner, cli_state, get_user_id_success, get_available_roles_success
+):
+    command = ["users", "remove-role", "test.username@example.com", "test"]
+    result = runner.invoke(cli, command, obj=cli_state)
+    assert result.exit_code == 1
+    assert "Role with name test not found." in result.output
```

Comment:
It might be clearer if we put quotes around the role name in the error so that the error message is 

`Role with name 'test' not found`